src/compiler/crystal/command.cr

```diff
@@ -159,8 +159,16 @@ class Crystal::Command
   private def init
     begin
```

Comment:
I mean this whole `begin`/`rescue` should be in `src/compiler/crystal/tools/init.cr`, probably inside the `Init.run` method.

Reply:
Thanks for the precision, should be good now!